#162397697 update readme file and build fix

samorahno opened the pull request:
#### What does this PR do?
This PR updates the README file to give it more meaning
#### Description of Task to be completed?
- Update the content of the README file
- fix the Travis build issue
#### How should this be manually tested?
- Clone this repository
- copy ```checkout ch-readmefile-update-162397697```
- View the ```README.md``` file and preview
- Should contain the updated content
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
162397697
#### Screenshots
![image](https://user-images.githubusercontent.com/45196374/49433201-b4d61080-f7b1-11e8-9911-256f1a58d272.png)
#### Questions
N/A

samorahno committed:
chore(readmeFile): update readme file

fob413 reviewed (changes requested):
You are not meant to push your coverage to github, please remove them.

samorahno committed:
chore(readmeFile): update the readme file

- update the readme file
- fix build issue
- update the gitignore file

[Finishes #162397697]
